recipe/meta.yaml

```
@@ -42,6 +42,7 @@ test:
     # - napari-omero  # [not linux]
   requires:
     - pip
```

```suggestion
    - pip
    - pyqt
```

Need to ensure a backend is present for the test

---

looks like this is actually problematic now.  omero-py requires Pillow>=10, but pyqt is apparently pulling in pillow 9?

I'm not entirely sure.  but honestly I'm inclined to just not test it here. it's got relatively little to do with this package directly